shared/constants/gas.js

```diff
@@ -39,6 +39,7 @@ export const PRIORITY_LEVELS = {
   HIGH: 'high',
   CUSTOM: 'custom',
   DAPP_SUGGESTED: 'dappSuggested',
+  ADVANCED: 'advanced',
 };
```

Comment:
`ADVANCED` is same as `CUSTOM`. Having 2 properties will complicate things.

Reply:
I reverted these changes;